webserver/Dockerfile

```
@@ -63,12 +63,13 @@ RUN cd /rw && composer install
 # https://github.com/NVIDIA/nvidia-docker/issues/714#issuecomment-386244620
 
 RUN cd /rw/extensions && git clone https://github.com/SemanticMediaWiki/SemanticCompoundQueries.git && cd SemanticCompoundQueries && git checkout f5eed72
-RUN cd /rw/extensions && git clone https://github.com/wikimedia/mediawiki-extensions-SemanticDrilldown SemanticDrilldown && cd SemanticDrilldown && git checkout a569de7
+RUN cd /rw/extensions && git clone https://github.com/AppChecker/mediawiki-extensions-SemanticDrilldown SemanticDrilldown && cd SemanticDrilldown && git checkout 17612a3
```

It seems like SemanticDrillDown probably now lives under the SemanticMediaWiki organization, and the old commit we know works seems like [it can be found there](https://github.com/SemanticMediaWiki/SemanticDrilldown/tree/a569de76012e29b465d1b4b84f91fa18bbb64803); I'd suggest we try to change as few things as possible at a time to make sure everything is still working.

```suggestion
RUN cd /rw/extensions && git clone https://github.com/SemanticMediaWiki/SemanticDrilldown SemanticDrilldown && cd SemanticDrilldown && git checkout a569de7
```